tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -72,8 +72,8 @@ execute: |
     LOOP="$(cat loop.txt)"
 
     echo "Run the snap-bootstrap tool"
-    go get ../../../lib/create-partitions
-    create-partitions \
+    go get ../../lib/uc20-create-partitions
+    uc20-create-partitions \
```

Comment:
Was this broken before?

Reply:
`create-partitions` was renamed to `uc20-create-partitions` in 0dc0dd6 but the test was not updated accordingly, causing a stale package error.